content/en/security/cloud_siem/_index.md

```diff
@@ -51,33 +51,126 @@ further_reading:
+
+Remain aware of any data processing issues or coverage gaps.
+
+- *Enabled Content Packs and Integrations:* View enabled content packs and integrations across the critical categories for providing comprehensive security coverage. Hover over each section of the horizontal bar to see which content packs are enabled in each category.
+- *Content Pack and Logs Health KPIs:* See whether any content packs or integrations are in warning or broken states so that you can quickly resolve any coverage gaps. Click the KPIs to view the affected content packs.
+- *Logs Analyzed by Cloud SIEM:* View logging trends across your top log sources and quickly identify any unusual spikes or drops. Click on the legend at the bottom to explore trends on a per source basis.
+
+### Important signals and cases
+
+{{< img src="security/security_monitoring/landing/02_important_signals_cases.png" alt="" style="width:100%;" >}}
+
+See important events happening in your environment, such as:
+
+- *Recent Open Signals Grouped By Rule:* Signals that are grouped by rule name and sorted by severity give you a quick overview of the most important signals in your environment. Click on the rows or on the severity pills to jump to a filtered view in the Signal Explorer for more details.
+- *Recent Open Security Cases:* Use [Case Management][5] for signals of interest that require additional analysis. View the current security cases in your environment and click on a row to see more details about the case.
+
+### Risk insights
+
+{{< img src="security/security_monitoring/landing/03_risk_insights.png" alt="" style="width:100%;" >}}
+
+Review the risky entities in your environment.
+
+- *Top Risky Entities:* Shows which entities have the highest risk scores at this time, and includes risk score, entity name, entity type, last risk score update timestamp, and risk score change. Click on an entity to view more details and take action.
+- *Entity Type Breakdown:* Shows which types of entities are most common in your environment. Click to view a list of entities filtered by that type.
+- *Entities risk score breakdown:* View entities by severity. Click to view the entities explorer with a filtered view of entities by severity.
+
+### Threat map
+
+{{< img src="security/security_monitoring/landing/04_threat_map.png" alt="" style="width:100%;" >}}
+
+Get insights from where the signals in your environment are getting generated.
+
+- *Top IPs by Country Distribution:* See which IPs are generating the most signals with a breakdown of important and less important signals. Also, use the map to see a list of signals by country.
+- *Signals by Country:* See the proportional breakdown from where signals are being generated. Click on the wedges to see signals by country and state/province, and spot signals being generated from unexpected countries.
+- *Signals by ISP Provider:* Review from which ISPs the signals are coming. Drill in by provider and location.
```

Comment:
"Drill in" is a phrase that we try to avoid: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/2752086538/Words+and+Phrases+to+Avoid#%E2%80%9Cdrill-down%E2%80%9D

```suggestion
- *Signals by ISP Provider:* Review from which ISPs the signals are coming. Click on a wedge on the pie chart to scope down by provider and location.
```